src/fetch.js

```
@@ -407,7 +407,7 @@ const useFetchComponentDescriptor = ({
   return _useFetch({
     fetchFunction: components.ocmComponent,
     fetchParams: params,
-    errorMessage: `Component "${componentName}" could not be fetched`,
+    showSnackbar: false,
```

do I understand it correctly that we do not omit the ocm requests for runetime artefacts, but just silence errors?

---

Yes, AFAIK it is not always the case that we don't have a correspondig OCM _component_ for a runtime _artefact_ (e.g. for Diki we indeed reference valid _components_ but just the artefact does not exist).